ButtonSet Docs: Adds `description` and `caption` for `ButtonSet` component (#834)
hashicorp/design-system: website/docs/components/dropdown/index.md

---
title: Dropdown

> description: Displays a list (typically of actions or links) through a toggle and is identifiable by the chevron icon in the button.

Reviewer: Would this be clearer do you think?
"Displays a list (typically of actions or links) revealed by a toggle button. Identifiable by the chevron icon in the button."

Author: Great idea! I'll change it.

> description: Displays a list (typically of actions or links) through a toggle and is identifiable by the chevron icon in the button.

Author: ```suggestion
description: Displays a list (typically of actions or links) revealed by a toggle button. Identifiable by the chevron icon in the button.
```
